setup-tdx-common

```diff
@@ -26,24 +26,25 @@
 # in case the packages in the PPA have older versions than the ones
 # from Ubuntu archive
 add_kobuk_ppa() {
```

Comment:
Could we create an other function add_kobuk_ppas() that will call add_kobuk_ppa ?

Reply:
Done